app/utils/affiliate.js

```diff
@@ -0,0 +1,50 @@
+import config from '../config/environment';
+
+function getServiceHost() {
+  if (config.environment === 'production') {
```

Comment:
you can use `fetch` service to get the env-specific service url (https://github.com/Wikia/ember-fandom/blob/master/addon/services/fetch.js). you will need to convert this util to the ember service